book/2_Selecting_an_AOI.md

```diff
@@ -0,0 +1,29 @@
+# Selecting an Area of Interest (AOI) for data analysis
+An `AOI` refers to the geographical extent of our study area, expressed in a specific co-ordinate reference frame [(CRS)](https://docs.qgis.org/3.28/en/docs/gentle_gis_introduction/coordinate_reference_systems.html#coordinate-reference-system-crs-in-detail).
```

Comment:
Should it be "coordinate reference system" instead of "co-ordinate reference frame"?

Reply:
Yep, my bad!

Reply:
Hi Karthik, what do you think to add some activities in the sections 3, 4 and 5. For example:

1. Getting Started
1. Selecting an AOI
1. Receiving data
**Activity A:** Choose a different area of interest, changing period and location, and then obtain data.
1. Analyzing datasets
**Activity B:** analyze the data from the new area of interest.
1. Manipulating and Visualizing datasets
**Activity C:** plot the flood map of the AOI.

Reply:
Hi Patri, I think that idea sounds good - in particular, "Activity A" makes sense as a method for learners to get used to the idea of changing the query to suit their needs.

Activity B and C are the default contents of the corresponding sections.

Reply:
I apologise; I made the comment in the wrong section. So I will update it.